### DISCUSSION TITLE
Can ICUB robots run a robot system using other PC motherboards?

### DISCUSSION THREAD

**Discussion by xilihutu888** [robotology/community#501] [Q&A] [answered]
Can the CCG012 and COMe-cKL6 motherboards of iCub robot head be replaced by ordinary computer motherboards? Can a regular computer motherboard run the latest system for iCub robots?

**Comment by pattacini** [comment, marked as answer]
>Can the CCG012 and COMe-cKL6 motherboards of iCub robot head be replaced by ordinary computer motherboards?

No, since the hub boards we mount on the iCub head are **low-power** and provide several **different I/O peripherals** to interface the robot devices, which are not available on standard PC motherboards.

Also, size matters in this context and PC motherboards do not have the required size to fit the head. 

>Can a regular computer motherboard run the latest system for iCub robots?

Yes, sure, as our SW ecosystem can run on many different platforms, including - especially I'd say - PC's.
Anyway, as said above, the essential factor preventing the installation of standard motherboards on the iCub head is the HW.

**Reply by xilihutu888** [reply]
Is there an alternative to the motherboard? I can't purchase CCG012 and COMe-cKL6 motherboards with such high cost at present. Perhaps you can tell me what I\ O port types ICUB robot motherboards need? Then I look for a suitable PC motherboard or industrial control motherboard to make the brain of ICUB robot. Thank you very much for your reply.

**Reply by pattacini** [reply]
I'm not into this, but I believe you should find what you're looking for by inspecting the schematics for the logic wiring available in our documentation at https://icub-tech-iit.github.io/documentation/icub_wiring/icub2_x/.

**Reply by xilihutu888** [reply]
Thank you very much for your reply, I have found what I want in the documentation.
